CheckYourEligibility-Parent/Views/Home/SchoolList.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@model CheckYourEligibility_FrontEnd.ViewModels.SchoolListViewModel
+@{
+    ViewData["Title"] = "Private Beta School List";
+    ViewData["Layout"] = "two_thirds";
+}
+<a class="govuk-back-link" href="@Url.Action("Index", "Home")" onclick="history.back(); return false;">Back</a>
+<script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
+@if (!ViewData.ModelState.IsValid)
+{
+    ViewData["Title"] = "Error: Check Details";
```

Comment:
Is this the correct title?

Reply:
I think probably not, copied over from an existing page and forgot to change, should probably say Error: School List ?